test/prob/std_normal/std_normal_test.hpp

```diff
@@ -0,0 +1,58 @@
+// Arguments: Doubles
+#include <stan/math/prim/scal.hpp>
+
+using std::vector;
+using std::numeric_limits;
+using stan::math::var;
+
+class AgradDistributionStdNormal : public AgradDistributionTest {
+public:
+  void valid_values(vector<vector<double> >& parameters,
+                    vector<double>& log_prob) {
+    vector<double> param(1);
+
+    param[0] = 0;           // y
+    parameters.push_back(param);
+    log_prob.push_back(-0.918938533204672669541); // expected log_prob
+
+    param[0] = 1;           // y
+    parameters.push_back(param);
+    log_prob.push_back(-1.418938533204672669541); // expected log_prob
+
+    param[0] = -2;          // y
+    parameters.push_back(param);
+    log_prob.push_back(-2.918938533204672669541); // expected log_prob
+  }
+ 
```

Comment:
Does this test NaN behavior?  If so, this should be sufficient.  I think it misses testing integer arguments to make sure they'll do the right thing, but we'll make sure they get instantiated properly elsewhere in Stan once this gets instantiated.

Reply:
Yes, it tests NaN behavior.